README.md

```diff
@@ -1,2 +1,7 @@
 # TextMining
 This is the base repo for the text mining and analysis project for Software Design, Spring 2016 at Olin College.
+
+
+#link to Project Write Up:
+# I hope this works, please let me know and I would be happy to fix it if it does not #link correctly
+/home/allison/Desktop/Link to Project3.pdf
```

Comment:
you can just use the Github link! [(ie)](https://github.com/allisonlynnbasore14/TextMining/blob/master/Project3.pdf). Also to put in a hyperlink in a markdown file (the .md files), you can use `[some text](the link)`